manifests/03-rbac-monitoring.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: monitoring-configmap-reader
```

Comment:
@s-urbaniak just to verify, this role doesn't already exist in some format that we should bind to instead.